ui/apps/everest/src/api/policies.ts

```diff
@@ -0,0 +1,14 @@
+import { api } from './api';
+import rbac from '/public/static/model.conf?raw';
```

Comment:
why we have "?" here

Reply:
https://vitejs.dev/guide/assets#importing-asset-as-string:~:text=Assets%20can%20be%20imported%20as%20strings%20using%20the%20%3Fraw%20suffix.